deploy/chart/catalog_resources/rh-operators/clusterloggings.crd.yaml

```diff
@@ -91,27 +101,46 @@ spec:
                     nodeSelector:
                       description: Define which Nodes the Pods are scheduled on.
                       type: object
-                    replicas:
+                    nodeCount:
                       description: Number of nodes to deploy for Elasticsearch
                       format: int32
                       type: integer
+                      minimum: 1
+                      default: 3
                     storage:
-                      description: 'The storage backing for Elasticsearch. More info: '
-                      type: object
+                      description: 'The storage specification for Elasticsearch data nodes'
+                      properties:
+                        size:
+                          description: 'The capacity of storage to provision.'
+                          type: string
+                          minimum: "10Gi"
```

Comment:
is this actually a fair minimum value? Is this an Elastic co. issued value?

Reply:
I don't think there is a defined minimum but Im guessing 200 is too large for say a 1 node dev cluster.  I dont really know what the balance should be.  Obviously if expect a large ES cluster, 10 node volumes will not cut it